### PR TITLE
Fixes/regression on pr 31

### PR DESCRIPTION
**Pull request by spuppala-rythmos**
Added fix for
Regression on PR 31 for switch case
and
Regression on PR 31 for cascading assignments

### CODE REVIEW

**Comment by Rassi** [comment]
We have found some regressions, and I have added tests for these: 60039 and 60040.

File: tests/staging.test [hunk 1]
```diff
@@ -1,55 +1,55 @@
-#
```
Comment (mihaipopescu) [inline tests/staging.test]: this looks like an EOL change

**Comment by spuppala-rythmos** [comment]
60039 and 60040 are working fine so moved them to regression.

**Review by hschlichter — APPROVED** [review]

**Comment by hschlichter** [comment]
test this please

**Pull request merged**